defaults/main.yml

```diff
@@ -107,7 +107,7 @@ rds_engine: mariadb
 rds_engine_version: '10.3'
 rds_instance_class: db.t3.micro
 rds_allocated_storage_gb: 20
-rds_storage_type: gp2 #  or "io1" for provisioned iops.
+rds_storage_type: gp3 #  or "io1" for provisioned iops.
```

Comment:
RDS doesn't support Gp3 yet, only EC2/EBS does. We'll have to stick with GP2 for the time being for RDS.

Reply:
@dale-c-anderson : Changed it back